### PR TITLE
Fixed react warnings.

### LINKED ISSUE #3289: Fix React warnings in src/app/containers/InlineContainer/index.test.jsx
**Describe the bug**
There are React warnings in unit tests.

**To Reproduce**
Go to simorgh in terminal
Run npm run test:unit
Observe React warnings:
```
  console.error src/testHelpers/setupTests.js:53
    Warning: Failed prop type: The prop `text` is marked as required in `InlineContainer`, but its value is `undefined`.
        in InlineContainer

  console.error src/testHelpers/setupTests.js:53
    Warning: Failed prop type: Invalid prop `blocks[0]` supplied to `InlineContainer`.
        in InlineContainer

  console.error src/testHelpers/setupTests.js:53
    Warning: Failed prop type: Invalid prop `blocks[0]` of type `array` supplied to `Blocks`, expected `object`.
        in Blocks (created by InlineContainer)
        in InlineContainer
```
**Expected behaviour**

No React warning for test suite.

**Testing notes**

No testing required.

### PR DESCRIPTION
**Pull request by hotinglok**
Resolves #3289 

**Overall change:**
Fixed react warnings.

**Code changes:**
- Added parameter to <InlineContainer/> when called to satisfy prop type warning. The ```text``` at top level can be different to that of ```blocks[0].model.text``` but it is currently not used which raises issue: 
- Removed extra brackets wrapping ```persianLink``` twice in ```InlineContainerWithContext```
- Added ids to blocks.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/InlineContainer/index.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const componentsToRender = { fragment, urlLink: InlineLink };
 
 const InlineContainer = ({ blocks, language }) => {
   const { script, service } = useContext(ServiceContext);
-
+  console.log('blocks', blocks[0]);
```
Comment (thekp) [inline src/app/containers/InlineContainer/index.jsx]:
```suggestion
```

File: src/app/models/propTypes/inline/index.js [hunk 2]
```diff
@@ -5,7 +5,6 @@ import { inlineLinkBlockPropTypes } from '../inlineLink';
 
 export const inlineModelPropTypes = {
   language: string.isRequired,
-  text: string.isRequired,
```
Comment (sareh) [inline src/app/models/propTypes/inline/index.js]: Text is required for the `aria-label` for the InlineLink. See here: https://github.com/bbc/simorgh/blob/latest/src/app/containers/InlineLink/index.jsx#L35

Comment (hotinglok) [reply src/app/models/propTypes/inline/index.js]: In that case, why is text not being used in the container if it is still being passed down to it?

Comment (dr3) [reply src/app/models/propTypes/inline/index.js]: @sareh src/app/containers/InlineContainer/index.jsx doesnt use `text` though, but it uses `inlineModelPropTypes` as a prop type, so it 'requires' a prop it never uses. Any idea whats wrong? Were not sure if the code is wrong, or the prop types

Comment (sareh) [reply src/app/models/propTypes/inline/index.js]: The `inlineModelPropTypes` are not complete. 
`inlineContainer/index.jsx` requires `blocks` - specifically `blocks[0].model.text` for the inlikeLink text for the aria-label.

Comment (dr3) [reply src/app/models/propTypes/inline/index.js]: This isnt `blocks[0].model.text` though, its just `text` at the top level (as far as I can see) :P. Is it needed at the top level outside of blocks?

Comment (hotinglok) [reply src/app/models/propTypes/inline/index.js]: Talked to sareh about this, confirmed that we're getting rid of `text` at the top level and not `blocks[0].model.text` so this is fine.

**Review by thekp — APPROVED** [review]

**Review by FK78 — APPROVED** [review]

File: src/app/containers/InlineContainer/__snapshots__/index.test.jsx.snap [hunk 3]
```diff
@@ -9,5 +27,13 @@ exports[`InlineContainer should render correctly 1`] = `
 <span
   className="c0"
   lang="fa"
-/>
+>
+  <a
+    aria-label="چیسربرگرundefined"
```
Comment (dr3) [inline src/app/containers/InlineContainer/__snapshots__/index.test.jsx.snap]: any ideas where this 'undefined' is coming from?

Comment (FK78) [reply src/app/containers/InlineContainer/__snapshots__/index.test.jsx.snap]: externalLinkText was missing.

File: src/app/models/propTypes/inline/index.js (hunk 2 above)
Comment (dr3) [inline src/app/models/propTypes/inline/index.js]: I think text is right here, e.g. this model https://github.com/bbc/simorgh/blob/ReactWarning3289/data/persian/articles/c19v65z3lg5o.json#L104-L116

**Review by dr3 — CHANGES_REQUESTED** [review]

**Review by chris-hinds — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

File: src/app/models/propTypes/inline/index.js (hunk 2 above)
Comment (dr3) [inline src/app/models/propTypes/inline/index.js]: Dont we want to keep this? Now that your passing it in anyway?

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/app/models/propTypes/inline/index.js (hunk 2 above)
Comment (dr3) [reply src/app/models/propTypes/inline/index.js]: Your passing text on https://github.com/bbc/simorgh/pull/3306/files#diff-1b925d2c9ac672c84c04aeeae7166d46R38 to satisfy the proptype, while also removing the prop type?

Comment (FK78) [reply src/app/models/propTypes/inline/index.js]: Yes.

**Review by dr3 — COMMENTED** [review]
LGTM

**Review by dr3 — APPROVED** [review]

**Pull request merged**